pom.xml

```diff
@@ -88,7 +88,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-war</artifactId>
-      <version>2.204</version>
+      <version>2.338-rc32148.f12eee9c85e2</version><!-- https://github.com/jenkinsci/jenkins/pull/5928 and https://github.com/jenkinsci/winstone/pull/174 -->
```

Comment:
Note that this means `jenkins-test-harness` and thus `plugin-pom` will require new `jenkins.version`s. Not necessarily a problem though (along with https://github.com/jenkinsci/plugin-pom/pull/478) it makes me reconsider my decision years ago to split these repos out of `jenkins`. It does mean that CloudBees CI development cannot pick up this fix for months. (CC @aneveux!)

The alternative is to have RJR detect the version of Winstone and fall back to the current “random & pray” algorithm for older versions, with a `TODO` comment to simplify in a year or so. Would make the logic harder to read and test, though.